packages/core/http/core-http-server/tsconfig.json

```diff
@@ -15,7 +15,8 @@
     "@kbn/utility-types",
     "@kbn/core-base-common",
     "@kbn/core-http-common",
-    "@kbn/zod"
+    "@kbn/zod",
+    "@kbn/core-http-router-server-internal"
```

Comment:
We can't have the public types depending on the internal types.
The way we've handled this before is to move shared dependencies to a common package.